index.js

```diff
@@ -73,8 +73,8 @@ function wrapInControlFlow(flow, globalFn, fnName) {
     function asyncTestFn(fn, description) {
       description = description ? ('("' + description + '")') : '';
       return function(done) {
-        var async = fn.length > 0;
-        testFn = fn.bind(this);
+        var async = fn.length > 0,
+          testFn = fn.bind(this);
```

Comment:
I prefer it that way too. Will change.
I only submitted it that way because I looked for other variable definitions and it seems there's no consistent style being used in this file.